src/views/TabView/TabBarBottom.js

```diff
@@ -53,11 +54,23 @@ type Props = {
   isLandscape: boolean,
 };
 
+type State = {
+  viewWidth: number,
+  maxTabItemWidth: number,
+};
+
 const majorVersion = parseInt(Platform.Version, 10);
 const isIos = Platform.OS === 'ios';
-const useHorizontalTabs = majorVersion >= 11 && isIos;
+const canUseHorizontalTabs = majorVersion >= 11 && isIos;
+const isPad =
+  Dimensions.get('window').height / Dimensions.get('window').width < 1.6;
```

Comment:
We avoid using `Dimensions` API in the library because `Dimensions` represents the layout of the whole app rather than the container. The navigator could be contained in a smaller container view and using the `Dimensions` API breaks that use case.

Reply:
I am using the Dimensions not as layout constraints, but as a device detection mechanism.  Knowing the device powers my heuristic-based layout approach (best-guess), and once the callback comes in I re-layout if necessary.  See my latest comment in the main thread for details.